packages/calcite-components/src/components/input-time-zone/input-time-zone.tsx

```diff
@@ -301,6 +304,7 @@ export class InputTimeZone
           disabled={this.disabled}
           label={this.messages.chooseTimeZone}
           lang={this.effectiveLocale}
+          maxItems={this.maxItems}
```

Comment:
how will this prop be used? What happens when more items exist than the max allowed items? Will items simply not render or will the customer receive some kind of warning/alert that they have exceeded the max items? Will this be included in a separate PR?

Reply:
Good questions. To summarize, this prop basically defines the amount of items displayed before a scrollbar is displayed.